File: modules/azule_entitlements.js
#!/usr/bin/env node

const fs = require('fs');
const plist = require('plist');
const args = process.argv.slice(2);

try {
  const data = fs.readFileSync(args[0], 'utf8');
  var json = plist.parse(data);
  for (let [i, val] of json['com.apple.security.application-groups'].entries()) {
    if (val.split('.').length > 4) {
      delete json['com.apple.security.application-groups'][i];
    }
  }
  fs.writeFileSync(args[0], plist.build(json));
}
catch {}

File: modules/azule_entitlements.py
#!/usr/bin/env python

import plistlib
import sys

file = open(sys.argv[1], 'rb')
data = plistlib.load(file)
key = 'com.apple.security.application-groups'

for val in list(data[key]):
	if len(val.split('.')) > 4:
		data[key].remove(val)

file = open(sys.argv[1], "wb")
plistlib.dump(data, file)